### DISCUSSION TITLE
Gatsby-Plugin-Sitemap custom sitemap index

### DISCUSSION THREAD

**Discussion by SimonKlausLudwig** [gatsbyjs/gatsby#27386] [Ideas / Feature Requests]
## Summary

Ability to add a custom sitemap index for gatsby-plugin-sitemap

### Basic example

I have a application with the following structure

mysite.de/en/ (English)
mysite.de/fr/ (French)
...

Each subfolder contains its own sitemap.xml. I want to add a sitemap index entry for the sitemap.xml in the root folder.
Is this possibly somehow ?

### Motivation

Even If i use alternate links https://support.google.com/webmasters/answer/189077 I think it makes sense to point to the sitemap.xml for each language.

**Comment by vladar** [comment]
@moonmeister What do you think about it?

**Comment by moonmeister** [comment]
Hmmm, for context @SimonKlausLudwig I'm in the middle of rewriting this plugin. How are you currently accomplishing the two different sitemaps? I haven't had this use case myself so hadn't thought through it much but if it means any significant changes to the plugin I want to do it now.

**Comment by SimonKlausLudwig** [comment]
I thought about to point to the language Sitemap.xml files via the Robots.txt or to put all the urls from the different languages within the root sitemap.xml. But as I asked I think the most elegant way is to add the language sitemap.xml to the roots sitemap-index. What do you think ?
